addon/components/o-s-s/text-area.ts

```diff
@@ -0,0 +1,46 @@
+import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+interface OSSTextAreaArgs {
+  rows?: number;
+  resize?: 'vertical' | 'horizontal' | 'both' | 'none';
```

Comment:
What happen when we use `both` as `resize`, I don't anything here. Add the case in `computedClass` method no?